### PR TITLE
Remove fec dns entries

### PR DESCRIPTION
**Pull request by jontours**
fec.gov is now being managed by the fec.  Keep wip for now, verifying if the open.fec.gov entries can be removed as well.

### CODE REVIEW

**Comment by jontours** [comment]
Let's ship it.

File: terraform/open.fec.gov.tf [hunk 1]
```diff
@@ -6,16 +6,10 @@ resource "aws_route53_zone" "open_fec_gov_zone" {
   }
 }
 
-resource "aws_route53_record" "open_fec_gov_open_fec_gov_a" {
-  zone_id = "${aws_route53_zone.open_fec_gov_zone.zone_id}"
-  name    = "open.fec.gov."
+resource "aws_route53_record" "open_fec_gov_open_fec_gov_a" {
```
Comment (mxplusb) [inline terraform/open.fec.gov.tf]: Why are we keeping this record and not clearing it out?

**Comment by mxplusb** [comment]
As a side bar, if you're actively working on a PR, you can always create a draft PR, which means it can't be merged until it's promoted to a "real" PR.

File: terraform/open.fec.gov.tf (hunk 1 above)
Comment (jontours) [reply terraform/open.fec.gov.tf]: FEC expressed some concerns about removing it.  But I think we can probably remove it.  Good catch.

**Pull request merged**